.github/workflows/main.yml

```diff
@@ -9,7 +9,7 @@ env:
 
 on:
   merge_group:
-  push:
+  pull_request:
```

Comment:
I think we'll still want push, but only on main? So that we run checks post-merge?

Or does the merge queue make that unessesary now?

so
```
 push:
   branches: [main]
```

Reply:
Hah, yep, that would be useful wouldn't it!